docs/introduction/licensedContent.mdx

```diff
@@ -0,0 +1,55 @@
+---
+title: Licensed Content
```

Comment:
thought we didnt include license related changes in 3.23, should we have this in documentations?